### PR TITLE
port: Fixing and refactoring integration tests

### PR DESCRIPTION
**Pull request by NicholasLYang**
### Description

This PR does a couple things:
- Standardizes test names to be `skeleton-case` and not `snake_case`
- Fixes the tests by using regexes for whitespace

### Testing Instructions

Closes TURBO-1608

### CODE REVIEW

**Review by mehulkar — COMMENTED** [review]
This is great, but I'd really like it to be split up. The file name changes and the `\s*` regex could probably be together since it's not changing behavior, but the actual output changes should be separate.

**Comment by NicholasLYang** [comment]
Yeah, sorry this PR got a little out of hand. Is it blocking though?

**Comment by mehulkar** [comment]
It's going to be a pain to merge this into #6279 so it would be pretty helpful to break it up, but I'll defer to you. I don't feel strongly enough to block other than "hey good git hygiene will be helpful in the long run"

**Comment by NicholasLYang** [comment]
Okay gotcha. I'm mostly asking because this feels like a PR that is unlikely to be reverted, but I'll split up the changes to make it a little easier on the merging side

File: turborepo-tests/integration/tests/dry-run.t
Comment (chris-olszewski) [inline turborepo-tests/integration/tests/dry-run.t:1]: Were there changes to this file or was it just a rename and GitHub is failing to recognize this?

File: turborepo-tests/integration/tests/find-turbo/hard-mode.t
Comment (chris-olszewski) [inline turborepo-tests/integration/tests/find-turbo/hard-mode.t:24]: Need to fixup the merge conflicts before this gets merged

**Review by chris-olszewski — CHANGES_REQUESTED** [review]

File: turborepo-tests/integration/tests/dry-run.t
Comment (NicholasLYang) [reply turborepo-tests/integration/tests/dry-run.t:1]: Just the trailing whitespace regex changes

**Review by chris-olszewski — APPROVED** [review]
Good to merge once integration tests pass. I think you just need to update the paths in the failing tests to use kebab case instead of snake

**Review by mehulkar — APPROVED** [review]
Thank you!

**Pull request merged**
